src/Mod/Path/Gui/Resources/panels/DogboneEdit.ui

```diff
@@ -111,7 +111,7 @@
        <item row="2" column="1">
         <widget class="QComboBox" name="incisionCombo">
          <property name="toolTip">
-          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Determines the incision length of the bone to be inserted into the profile.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;adaptive&lt;/span&gt; ... the lenght is adapted to cover the corner based on the angle of it's edges, taking the current tool radius into account (default)&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;fixed&lt;/span&gt; ... is the same as adaptive for straight angles. For T-bones it's the radius of the tool (R) and for dogbones it's R * (2/√2 - 1).&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;custom&lt;/span&gt; ... let's you sepcify a custom (fixed) length below&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
typo here was `lenght`. for some reason making the change affected the whole contents of `<string>` instead of just the the single word.